src/EIPTestsFiller/StateTests/stEOF/stCREATE3/CREATE3Filler.yml

```diff
@@ -0,0 +1,76 @@
+CREATE3:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 200000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul 
+        {
+          let aux_data_size := calldatasize()
+          calldatacopy(0, 0, aux_data_size)
+          sstore(0, call(gas(), 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0, 0, aux_data_size, aux_data_size, 32))
+          sstore(1, mload(aux_data_size))
+          sstore(2, 1)
+        }
+      nonce: 0
+      storage: {}
+    c94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      # Outer container code: 
+      #   calldatacopy(0, 0, calldatasize())
+      #   create3{0}(0, 0, calldatasize(), 0xff)
+      #   mstore(0)
+      #   return(0, 32)
+      #
+      #   Embedded container 0 (inticontainer) code:
+      #     calldatacopy(0, 0, calldatasize())
+      #     returncontract{0}(0, calldatasize())
+      #
+      #     Embedded container 0 (deployed container):
+      #       code: "0xfe"
+      #       data: "0xabcdef"
+      code: '0xef00010100040200010017040001003a030000000000000436600060003760ff3660006000f60060005260206000f3ef0001010004020001000b04000100170300000000000003366000600037366000f700ef000101000402000100010300030000000000feabcdef'
```

Comment:
First Embedded container has `CREATE4` (`0xf7`) instead of `RETURNCONTRACT` (`0xf8`):

```
EF0001 # Magic and Version ( 1 )
010004 # Types length ( 4 )
020001 # Total code sections ( 1 )
  000b # Code section  0 , 11  bytes
030000 # Data section length ( 0 )
040001 # Total container sections ( 1 )
  0017 # Container section  0 , 23  bytes
    00 # Terminator (end of header)
       # Code 0 types
    00 # 0 inputs
    00 # 0 outputs
  0003 # max stack: 3
       # Code section 0
    36 # CALLDATASIZE 
  6000 # PUSH1(0) 
  6000 # PUSH1(0) 
    37 # CALLDATACOPY 
    36 # CALLDATASIZE 
  6000 # PUSH1(0) 
    f7 # CREATE4 
    00 # STOP 
       # Data section (empty)
       # Container section 0
ef000101000402000100010300030000000000feabcdef
```